### PR TITLE
iPhone 4 compatibility

### PR DESCRIPTION
**Pull request by taavirehemagi**
The problem of projects screen not opening:
http://stackoverflow.com/questions/18746929/using-auto-layout-in-uitableview-for-dynamic-cell-layouts-variable-row-heights/18746930#18746930

The problem of workspace selector crashing the app is that UIPopoverPresentationController is not present in iOS 7, so check for a class first. If no class present, +1 the currentWorkspaceIndex #quickfix

### CODE REVIEW

File: Ross/ViewControllers/ProjectSelectionViewController.cs [hunk 1]
```diff
@@ -119,13 +119,13 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
                 if (data is ProjectData)
                 {
-                    var cell = (ProjectCell)tableView.DequeueReusableCell(ProjectCellId, indexPath);
+                    var cell = (ProjectCell)tableView.DequeueReusableCell(ProjectCellId);
```
Comment (monday8am) [inline Ross/ViewControllers/ProjectSelectionViewController.cs]: why these changes? Add a comment or a link to some explanation.

File: Ross/ViewControllers/ProjectSelectionViewController.cs [hunk 2]
```diff
@@ -97,8 +97,22 @@ private void OnAddNewProject(object sender, EventArgs evt)
...
+
+            bool hasPopover = ObjCRuntime.Class.GetHandle("UIPopoverPresentationController") != IntPtr.Zero;
+            if (hasPopover)
+            {
+                var popoverController = new WorkspaceSelectorPopover(viewModel, sourceRect);
+                PresentViewController(popoverController, true, null);
+            }
+            else
+            {
+                var nextWorkspace = viewModel.CurrentWorkspaceIndex + 1;
```
Comment (monday8am) [inline Ross/ViewControllers/ProjectSelectionViewController.cs]: seems to be you iterate over the list of workspaces. Add a comment please.

Comment (taavirehemagi) [inline Ross/ViewControllers/ProjectSelectionViewController.cs]: because we cannot show the picker for earlier iOS version, I had to come up with an alternative way to change between workspaces in iOS 7. it goes to the next workspace with every click now. (works the same in later versions)

File: Ross/ViewControllers/ProjectSelectionViewController.cs (hunk 1 above)
Comment (taavirehemagi) [inline Ross/ViewControllers/ProjectSelectionViewController.cs]: I can't find the link at the moment but if I remember correctly, the function with the indexPath was introduced a bit later and might not be supported in earlier version.
